src/Raven.Server/Documents/Replication/ConflictManager.cs

```diff
@@ -67,12 +67,20 @@ public ConflictManager(DocumentDatabase database, ResolveConflictOnReplicationCo
                     Flags = flags
                 };
 
+                Exception err = null;
                 if (IsSameCollection(documentsContext, id, conflictedDoc.Collection))
                 {
-                    if (TryResolveConflictByScript(
-                        documentsContext,
-                        conflictedDoc))
-                        return;
+                    try
+                    {
+                        if (TryResolveConflictByScript(
```

Comment:
I would prefer to return `false` instead of catch, capture and rethrowing the exception.

Reply:
But we do need to alert the user about this by raising a notification in the notification center

Reply:
Done